### PR TITLE
CLOUDP-108870: Retrieve a specific private endpoint interface

### PR DESCRIPTION
**Pull request by johansteyn**
## Proposed changes

_Jira ticket:_ CLOUDP-108870

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

I "smoke-tested" this by creating a GCP private endpoint in Atlas and running the command:

```
% bin/mongocli atlas privateEndpoints gcp interfaces describe endpoint-1 --endpointServiceId 61eaca605af86411903de1dd
ENDPOINT     STATUS      DELETE REQUESTED
endpoint-1   AVAILABLE   false
```

### CODE REVIEW

File: internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+// mongocli atlas privateEndpoint(s) gcp interface(s) describe <endpointGroupId> --endpointServiceId endpointServiceId [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe <endpointGroupId>",
+		Aliases: []string{"get"},
+		Args:    require.ExactArgs(1),
+		Short:   "Return a specific GCP private endpoint interface for your project.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			opts.privateEndpointGroupID = args[0]
```
Comment (andreaangiolillo) [inline internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go:68]: you need to add `Annotations` and `Example`. See this command for an example https://github.com/mongodb/mongocli/blob/8247c6f0a5d45ecba016a79cf73459eecadeb227/internal/cli/atlas/privateendpoints/gcp/delete.go#L59

File: internal/cli/atlas/privateendpoints/gcp/interfaces/describe_test.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 MongoDB Inc
+//
```
Comment (andreaangiolillo) [inline internal/cli/atlas/privateendpoints/gcp/interfaces/describe_test.go:2]:
```suggestion
// Copyright 2022 MongoDB Inc
//
```

File: internal/cli/atlas/privateendpoints/gcp/interfaces/interfaces_test.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 MongoDB Inc
+//
```
Comment (andreaangiolillo) [inline internal/cli/atlas/privateendpoints/gcp/interfaces/interfaces_test.go:2]:
```suggestion
// Copyright 2022 MongoDB Inc
//
```

**Review by andreaangiolillo — COMMENTED** [review]
Great works! Just a few comments, nothing major

File: internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go (hunk 1 above)
Comment (johansteyn) [reply internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go]: Thanks, I've added those now.

File: internal/cli/atlas/privateendpoints/gcp/interfaces/describe_test.go (hunk 2 above)
Comment (johansteyn) [reply internal/cli/atlas/privateendpoints/gcp/interfaces/describe_test.go]: Fixed.

File: internal/cli/atlas/privateendpoints/gcp/interfaces/interfaces_test.go (hunk 3 above)
Comment (johansteyn) [reply internal/cli/atlas/privateendpoints/gcp/interfaces/interfaces_test.go]: Fixed.

File: internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe <endpointGroupId>",
+		Aliases: []string{"get"},
+		Args:    require.ExactArgs(1),
+		Short:   "Return a specific GCP private endpoint interface for your project.",
+		Annotations: map[string]string{
+			"args":                "endpointGroupId",
+			"requiredArgs":        "endpointGroupId",
+			"endpointGroupIdDesc": "Unique identifier for the endpoint group.",
```
Comment (JuliaMongo) [inline internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go:70]: Please see this [description](https://github.com/mongodb/mongocli/pull/921/files#diff-12aba3cdbb338fc867f81690686551f33a06c96c549f29f1b859637f8e26bf82R59-R60) -- shouldn't they all match?

**Review by JuliaMongo — CHANGES_REQUESTED** [review]

File: internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go (hunk 4 above)
Comment (johansteyn) [reply internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go:70]: No, they are not the same and should not match, Julia.
The arg you are referring to is a private endpoint ID that is specific to AWS.
Here we are using something different: an endpoint group ID that is specific to GCP.

Comment (JuliaMongo) [reply internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go:70]: @johansteyn if so, shouldn't we still use the same style for the description, and indicate the number or chars?

**Review by andreaangiolillo — APPROVED** [review]

File: internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go (hunk 4 above)
Comment (johansteyn) [reply internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go:70]: @JuliaMongo Adding to what I said: when creating a GCP endpoint in the Atlas UI, there are no restrictions on the "Private Service Connect Endpoint Prefix" field, other than that it be unique. This field is what is used as the "endpointGroupId".

Comment (JuliaMongo) [reply internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go:70]: I should have checked the docs for this myself and thank you for your patience. Agreed now!

**Pull request merged**